package.json

```diff
@@ -54,24 +58,22 @@
         "build:extension": "npm run esbuild:extension && npm run esbuild:ls",
         "esbuild:extension": "esbuild ./src/extension.ts --bundle --outfile=out/extension.js --external:vscode --format=cjs --platform=node",
         "esbuild:ls": "esbuild ./src/language-server/main.ts --bundle --outfile=out/language-server/main.js --format=cjs --platform=node",
-        "build:web": "npm run build && npm run prepare:public && npm run build:worker && npm run copy:monaco-editor-wrapper && npm run copy:monaco-workers",
+        "build:web": "npm run build && npm run prepare:public && npm run build:worker && node scripts/copy-monaco-assets.mjs",
         "build": "tsc -b tsconfig.json",
```

Comment:
This is **compile** and not **build** 😉

Reply:
By convention **build** is used to build everything. This is especially helpful in mono-repos/workspaces.

Reply:
It's both, the primary build target is to compile & build the parser & cli. So in a mono-repo this is what you would likely want (minilogo parser, generator, cli, etc.) to use in a larger project.

The web aspect, although prominent, is more of an additional build target (consuming the compiled results of the first). It's not what you would typically be consuming in another project (at least that's not what I was intending to set this up as).

If that's not too clear maybe it would be good to clarify this in the README in further detail then?

Reply:
No, it's alright 🙂